frappe/__init__.py

```diff
@@ -59,6 +59,14 @@
 	warnings.simplefilter("always", DeprecationWarning)
 	warnings.simplefilter("always", PendingDeprecationWarning)
 
+# Always initialize sentry SDK if the DSN is sent
+if sentry_dsn := os.getenv("FRAPPE_SENTRY_DSN"):
+	import sentry_sdk
+
+	from frappe.utils.sentry import before_send
+
+	sentry_sdk.init(dsn=sentry_dsn, before_send=before_send, release=__version__)
```

Comment:
Should we also setup [tracing](https://docs.sentry.io/platforms/python/performance) here and allow that to be configured? Or is that planned for a future iteration?

We've gained some insights into backend performance from it (one of the projects below):

![image](https://github.com/frappe/frappe/assets/13396535/f986448a-744c-492e-be75-d56761931511)

Reply:
We have enabled tracing internally but we'll likely add it as an optional feature in future iterations. First cut is just for error reports.